targets/unity-v2/source/Authentication/Strategies/EmailPasswordAuthStrategy.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using PlayFab.ClientModels;
+using UnityEngine;
+
+namespace PlayFab.Authentication.Strategies
+{
+    internal sealed class EmailPasswordAuthStrategy : IAuthenticationStrategy
+    {
+        public void Authenticate(PlayFabAuthService authService, Action<LoginResult> resultCallback, Action<PlayFabError> errorCallback)
+        {
+            if (authService.RememberMe && !string.IsNullOrEmpty(authService.RememberMeId))
+            {
+                PlayFabClientAPI.LoginWithCustomID(new LoginWithCustomIDRequest
+                {
+                    TitleId = PlayFabSettings.TitleId,
+                    CustomId = authService.RememberMeId,
+                    CreateAccount = true,
+                    InfoRequestParameters = authService.InfoRequestParams
+                }, resultCallback, errorCallback);
+
+                return;
+            }
+
+            // a good catch: If username & password is empty, then do not continue, and Call back to Authentication UI Display
```

Comment:
I agree it's a good catch, but please don't praise yourself in your comments. Haha.
Keep the comment drop the praise.